test/e2e/testdata/docker_registry_secret.yaml

```diff
@@ -0,0 +1,22 @@
+#Licensed to the Apache Software Foundation (ASF) under one
+#or more contributor license agreements.  See the NOTICE file
+#distributed with this work for additional information
+#regarding copyright ownership.  The ASF licenses this file
+#to you under the Apache License, Version 2.0 (the
+#"License"); you may not use this file except in compliance
+#with the License.  You may obtain a copy of the License at
+#
+#http://www.apache.org/licenses/LICENSE-2.0
+#
+#Unless required by applicable law or agreed to in writing, software
+#distributed under the License is distributed on an "AS IS" BASIS,
+#WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+#See the License for the specific language governing permissions and
+#limitations under the License.
+apiVersion: v1
+data:
+  .dockerconfigjson: eyJhdXRocyI6eyJkb2NrZXItZGV2LnJlcG9zaXRvcnkuY2xvdWRlcmEuY29tIjp7InVzZXJuYW1lIjoiZG9ja2VyLWRldi1yZWFkZXIiLCJwYXNzd29yZCI6IjFWVlEzbkVLM3BxMXdqVEoiLCJlbWFpbCI6ImFwYXRoYW5AY2xvdWRlcmEuY29tIiwiYXV0aCI6IlpHOWphMlZ5TFdSbGRpMXlaV0ZrWlhJNk1WWldVVE51UlVzemNIRXhkMnBVU2c9PSJ9fX0=
```

Comment:
What kind of a secret is this? Is it safe to expose like that?

Reply:
Definitely not safe - this contains credentials for a user to a private docker repository. This needs to be removed and the credentials considered compromised globally.

Reply:
cc @manirajv06